rts/lra/test/README.adoc

```diff
@@ -1,57 +1,63 @@
+with maven plugin to start the runtime and deploy lra-coordinator.
+
 The `lra/test` module contains the following submodules
 
-* link:./arquillian-extension/[`arquillian-extension`] contains set of
-  Arquillian extension classes which makes possible to execute the test modules
-  on supported runtimes
+* link:./arquillian-extension/[`arquillian-extension`] contains Arquillian extension classes
+  which make possible to execute the test modules on supported runtimes
 * link:./basic[`basic`] contains integration tests which are written for testing
   the Narayana features of the
   https://github.com/eclipse/microprofile-lra[LRA specification]
-* link:./tck[`tck`] is a module which is responsible for execution of
+* link:./tck[`tck`] is a module which is responsible for the execution of
   https://github.com/eclipse/microprofile-lra/tree/master/tck[LRA TCK tests]
   with Narayana supported runtimes
-
-All tests are executed with help of https://arquillian.org[Arquillian] platform.
-Arquillian is responsible to launch the runtime and makes possible to execute
-the tests.
+* link:./crash[`crash`] hosts tests that simulate a crash of either lra-coordinator or
+  lra-participant or both. So far, only Wildfly is considered in this module as Quarkus
+  does not have an Arquillian adapter to run unmanaged tests
 
 == How to run integration tests
 
-The itegration tests which should be run on the application runtime
-are configured to be started with https://maven.apache.org/surefire/maven-failsafe-plugin/[Maven Failsafe Plugin].
+Integration tests are tested with
+https://maven.apache.org/surefire/maven-failsafe-plugin/[Maven Failsafe Plugin], using the target 'verify'.
 
-To run the tests the target `verify` has to be used.
-Single test with `-Dit.test=...`.
-Debug the Maven test execution with `-Dmaven.failsafe.debug`.
+Single test can be executed with `-Dit.test=...`
+To use Failsafe in debug mode, use `-Dmaven.failsafe.debug`
 
-The Failsafe Plugin link:../pom.xml#114[is configured] to run tests with names
+The Failsafe Plugin link:../pom.xml[is configured] to run tests with names
 `*IT.java`, `*Tests.java` and `*TestCase.java`.
 
 === Application runtimes
 
 The module is configured to run two runtimes - https://www.wildfly.org[WildFly]
-and link:https://quarkus.io[Quarkus]/link:https://thorntail.io[Thorntail].
+and link:https://quarkus.io[Quarkus].
 
-The **WildFly** runtime deploys the coordinator and the application with the client
-library at the same application server. The coordinator is deployed as
-a separated `war` archive, and the same the applications which packs together
-the LRA client library.
+When the **WildFly** runtime is used (through the 'arq' profile), two instances of Wildfly are initiated:
+ * The first instance is started using WildFly Maven Plugin, then lra-coordinator is deployed using the dedicated WAR
+ * The second instance is started using Arquillian, then an lra-participant is deployed
```

Comment:
What is `lra-participant`, I cannot find any other reference to this?

Reply:
`lra-participant`s are JAX-RS applications implementing the LRA specification. IIRC, `participant` is an actor defined in the MP LRA specification

Reply:
The question is not only for my benefit, it is for the benefit of any community member that wants to run our test suite.

Reply:
I'm still not clear about what `lra-participant` and `lra-coordinator` mean.

Reply:
Are they module names or artifacts names or property names, something else?

Reply:
> The model concerns participants (aka compensators), a logical coordinator and a client. A client starts a new LRA via an annotation which results in the creation of a new LRA. When a business service does work that may have to be later compensated it enlists a participant with the LRA which, when the client later closes or cancels the LRA, operates on behalf of the service to undo the work the service performed within the scope of an LRA or to compensate for the fact that the original work could not be completed.

from [here](https://github.com/eclipse/microprofile-lra/blob/master/spec/src/main/asciidoc/microprofile-lra-spec.adoc)

From the LRA spec, I understood that `lra-coordinator` is the logical coordinator, `lra-participant` is the compensator (aka the application implementing the LRA APIs) and the client is the service that makes the calls. Please, let me know if I totally misunderstood this

Reply:
Yes I think you have. `lra-coordinator` and `lra-participant` look like a variable names or artifacts etc, i.e. something that a computer would parse. The specification is written using words that mostly can be looked up in a dictionary so something like LRA coordinator or LRA participant is probably what you meant.

Reply:
Thanks Michael, I am gonna clarify the wording so that the community does not get confused